### PR TITLE
Fixes #3198 fix eslint for root, test, backend, satellite

### LINKED ISSUE #3198: Configure ESLint for root, test, backend
`test/*` needs to be configured to add the new `@senecacdot/eslint-config-telescope` to `devDependencies` and configure `ESLint` to actually work. We added an `.eslintrc` file in #3134 to `test/` but it is currently not being read anywhere.

Adding eslint in root should lint `test/` and `src/backend`. Once the backend has been moved to its own microservice, we won't need eslint in root anymore.

### PR DESCRIPTION
**Pull request by cindyledev**
## Issue This PR Addresses

Fixes #3198

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR adds eslint scripts and eslint dependencies so running `pnpm lint` in root will lint files in the following directories:

- src/backend
- test

There is also one file that I missed while fixing the eslint for Satellite in #3166

## Steps to test the PR

1. Add my repository as a remote: `git remote add cindyledev https://github.com/cindyledev/telescope.git`
2. Fetch my branches: `git fetch cindyledev`
3. Checkout my branch: `git checkout issue-3198`
4. Install dependencies using pnpm in root: `pnpm install`
5. Run `pnpm lint`

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by cindyledev** [comment]
Whoops, will fix unit test after dinner :)

**Review by humphd — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -92,11 +94,13 @@
     "@babel/preset-env": "7.16.11",
     "@babel/preset-react": "7.16.7",
     "@babel/preset-typescript": "7.16.7",
+    "@senecacdot/eslint-config-telescope": "latest",
```
Comment (tcvan0707) [inline package.json:97]: Why do we need the `latest` version in here? Can't we specify the version of `@senecacdot/eslint-config-telescope`?

**Review by tcvan0707 — COMMENTED** [review]
Good job @cindyledev. I have a question about the version of `@senecacdot/eslint-config-telescope`, see my comment below.

File: package.json (hunk 1 above)
Comment (cindyledev) [reply package.json:97]: I copied and pasted this line from somewhere in our code. But you're right, it should be 1.0.0. I will fix this by tomorrow morning. Thanks for catching this!

**Review by tcvan0707 — APPROVED** [review]

**Review by aserputov — APPROVED** [review]

**Pull request merged**